doc/release/1.9.0-notes.rst

```diff
@@ -152,14 +391,674 @@ For more information on the introduction of Meson support in SciPy, see
+* Evgeni Burovski (96)
+* Matthias Bussonnier (20)
+* Dominic C (1)
+* Cameron (1) +
+* CJ Carey (3)
+* Thomas A Caswell (2)
+* Ali Cetin (2) +
+* Hood Chatham (4) +
+* Klesk Chonkin (1)
+* Craig Citro (1) +
+* Dan Cogswell (1) +
+* Luigi Cruz (1) +
+* Anirudh Dagar (5)
+* Brandon David (1)
+* deepakdinesh1123 (1) +
+* Denton DeLoss (1) +
+* derbuihan (2) +
+* Sameer Deshmukh (13) +
+* Niels Doucet (1) +
+* DWesl (8)
+* eytanadler (30) +
+* Thomas J. Fan (5)
+* Isuru Fernando (3)
+* Joseph Fox-Rabinovitz (1)
+* Ryan Gibson (4) +
+* Ralf Gommers (302)
+* Srinivas Gorur-Shandilya (1) +
+* Alex Griffing (2)
+* h-vetinari (3)
+* Matt Haberland (435)
+* Tristan Hearn (1) +
+* Jonathan Helgert (1) +
+* Samuel Hinton (1) +
+* Jake (1) +
+* Stewart Jamieson (1) +
+* Jan-Hendrik Müller (1)
+* Yikun Jiang (1) +
+* JuliaMelle01 (1) +
+* jyuv (12) +
+* Chris Keefe (1) +
+* Robert Kern (4)
+* Andrew Knyazev (11)
+* Matthias Koeppe (4) +
+* Sergey Koposov (1)
+* Volodymyr Kozachynskyi (1) +
+* Yotaro Kubo (2) +
+* Jacob Lapenna (1) +
+* Peter Mahler Larsen (8)
+* Eric Larson (4)
+* Laurynas Mikšys (1) +
+* Gregory R. Lee (2)
+* lerichi (1) +
+* Tim Leslie (2)
+* P. L. Lim (1)
+* Smit Lunagariya (43)
+* lutefiskhotdish (1) +
+* Cong Ma (12)
+* Syrtis Major (1)
+* Nicholas McKibben (17)
+* Melissa Weber Mendonça (10)
+* Mark Mikofski (1)
+* Jarrod Millman (13)
+* Harsh Mishra (6)
+* ML-Nielsen (3) +
+* Matthew Murray (1) +
+* Andrew Nelson (50)
+* Dimitri Papadopoulos Orfanos (1) +
+* Evgueni Ovtchinnikov (2) +
+* Sambit Panda (1)
+* Nick Papior (2)
+* Tirth Patel (43)
+* Petar Mlinarić (1)
+* petroselo (1) +
+* Ilhan Polat (64)
+* Anthony Polloreno (1)
+* Amit Portnoy (1) +
+* Quentin Barthélemy (9)
+* Patrick N. Raanes (1) +
+* Tyler Reddy (93)
+* Pamphile Roy (192)
+* Vivek Roy (2) +
+* Niyas Sait (2) +
+* Atsushi Sakai (25)
+* Mazen Sayed (1) +
+* Eduardo Schettino (5) +
+* Daniel Schmitz (6) +
+* Eli Schwartz (3) +
+* SELEE (2) +
+* Namami Shanker (4)
+* siddhantwahal (1) +
+* Gagandeep Singh (8)
+* Soph (1) +
+* Shivnaren Srinivasan (1) +
+* Scott Staniewicz (1) +
+* Leo C. Stein (4)
+* Albert Steppi (7)
+* Christopher Strickland (1) +
+* Kai Striega (4)
+* Søren Fuglede Jørgensen (1)
+* Aleksandr Tagilov (1) +
+* Masayuki Takagi (1) +
+* Sai Teja (1) +
+* Ewout ter Hoeven (1) +
+* Will Tirone (2)
+* Bas van Beek (7)
+* Dhruv Vats (1)
+* H. Vetinari (4)
+* Arthur Volant (1)
+* Samuel Wallan (5)
+* Stefan van der Walt (8)
+* Warren Weckesser (81)
+* Anreas Weh (1)
+* Nils Werner (1)
+* Aviv Yaish (1) +
+* Dowon Yi (1)
+* Rory Yorke (1)
+* Yosshi999 (1) +
+* yuanx749 (2) +
+* Gang Zhao (23)
+* ZhihuiChen0903 (1)
+* Pavel Zun (1) +
+* David Zwicker (1) +
+
+A total of 152 people contributed to this release.
+People with a "+" by their names contributed a patch for the first time.
+This list of names is automatically generated, and may not be fully complete.
 
 
 ***********************
 Issues closed for 1.9.0
 ***********************
```

Comment:
Indeed, see `tools/gh_lists.py`.

Grabbing issues based on date of closure isn't going to work, because a lot are closed because fixed in older releases, invalid reports, etc.

As of now, you should set milestones on issues that you close. A way to improve the script could be to parse commit messages for "Closes xxx", but even that isn't going to be foolproof since I often see that people fail to add that info to commit messages and only add it to PR descriptions.

Reply:
A possible automation could be to auto-add a milestone to an issue which is closed by a PR  with a milestone. Maybe that's a feature request to github or maybe it's possible to do with their api.

Reply:
I started to do this for issues I was involved with. I thought I should make sure I'm doing the right thing before going further. For instance, I added the milestone to gh-14622 because it was a legitimate issue that was closed by a PR, whereas I did not add a milestone to gh-15160 because it was reported due to a misunderstanding. LMK if I should keep going this way.

Reply:
Yes, that sounds right, thanks Matt!

Reply:
OK, done. I added a lot of milestones. 
I accidentally created a 1.90 milestone in the process. Do you know how to remove that from the list?

Reply:
Thanks Matt. I deleted the 1.90 milestone.